### PR TITLE
Refactored out const fields

### PR DESCRIPTION
**Pull request by maxqchen**
In preparation for the serialization step.

And I'm pretty sure someone will ask, so:
Interface fields have implicit public static final modifiers and this is a common pattern for declaring constants.

### CODE REVIEW

**Comment by maxqchen** [comment]
Weird thing: on my repo it says " This branch is 1 commit ahead, 24 commits behind manifold-lang:master " even though the PR here doesn't seem to have any issues with it. I did a fetch from manifold/master and it didn't seem like my stuff was behind. Is it just GH being late on updating that status?

**Comment by mtrberzi** [comment]
LGTM, ship it!

File: src/main/java/org/manifold/compiler/middle/serialization/SeralizationConsts.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package org.manifold.compiler.middle.serialization;
+
+public interface SeralizationConsts {
```
Comment (TysonAndre) [inline src/main/java/org/manifold/compiler/middle/serialization/SeralizationConsts.java]: Should be spelled `SerializationConsts`

**Comment by TysonAndre** [comment]
LGTM after typo fixed.

This could also be done with https://en.wikipedia.org/wiki/Static_import from a final class, since it's entirely for constants, but that's mostly nitpicking.

**Comment by maxqchen** [comment]
Been traveling around and stuff, kinda dropped this. Changed that 1 typo Tyson pointed out, but not sure why the diff decided to include the entire file. The white space even seems to be the same but git doesn't think it is? Using git diff with the -b flag shows the actual changes.

**Comment by mtrberzi** [comment]
Nice, looks good. The diff stats are always so weird. I'm sure our commit scores are all artificially inflated by now because of the fact that we did all that refactoring last term...

**Pull request merged**